torch/_inductor/ir.py

```diff
@@ -3622,14 +3622,17 @@ def codegen_kwargs(self):
         if V.graph.cpp_wrapper:
             # FIXME: we should unconditionally fill self.kwargs with missing default values
             # instead of carrying an extra self.ordered_kwargs_for_cpp_kernel
-            if self.kwargs:
+            if self.kwargs and not self.ordered_kwargs_for_cpp_kernel:
                 assert (
                     self.ordered_kwargs_for_cpp_kernel
                 ), "ordered_kwargs_for_cpp_kernel is missing"
```

Comment:
Looks like the assert condition is always False? Probably we could just thrown an exception here?